app/scripts/controllers/crud-base-controller.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+/**
+ * @ngdoc function
```

Comment:
A whole lot of Nickleback right here. [https://i.ytimg.com/vi/5abFEQ0dKwk/maxresdefault.jpg](url)